regression/cbmc-incr-oneloop/alarm2/trace_test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--incremental-loop main.0 --unwind-min 5 --unwind-max 10 --trace
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+^State \d+ file main.c function havocIO line 501 thread \d+$
+--
+^warning: ignoring
```

Comment:
A description of why you added this test (e.g. tests the trace production for ??)